charts/shoot-core/components/charts/cluster-identity/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Why we have duplicated the same chart three times? Probably it makes sense to have it only once somewhere in `charts`. It's always the same.

Reply:
This comment is not yet resolved. We are still having the chart multiple times.

Reply:
The chart for the Shoot cluster identity is part of the "shoot-core" managed resources. Should I create new managed resource or change the logic how the shoot-core is calculated?

Reply:
No, you could simply symlink the chart into `shoot-core` to avoid duplication.

Reply:
Ah yes, that will work, thanks!